HabitLogger.gkemeza/HabitLogger.gkemeza/Methods.cs

```diff
@@ -0,0 +1,595 @@
+﻿using Microsoft.Data.Sqlite;
+using System.Globalization;
+
+namespace HabitLogger
+{
+    internal class Methods
+    {
+        Program program = new Program();
+
+        public void CreateTwoTables()
+        {
+            string connectionSource = @"Data Source=HabitLogger.db";
```

Comment:
The `connectionSource` is declared in every method. Now let's say your boss wants to change the database file name. Now you need to update it in multiple places and hope you do it in all places correctly. My suggestion is to try and become as lazy as possible and have only one instance variable with the `connectionSource` 😉